components/Alias.tsx

```diff
@@ -0,0 +1,30 @@
+import { useContext, useMemo } from "react";
+import { AppStateContext } from "../context/state";
+
+const Alias = ({
+  address,
+  className,
+}: {
+  address: string;
+  className?: string;
+}) => {
+  const state = useContext(AppStateContext)!;
+
+  const formatted = useMemo(
+    () =>
+      `${address.substring(0, 5)}...${address.substring(address.length - 5)}`,
+    [address]
+  );
+
+  const toDisplay = useMemo(
+    () =>
+      state.aliases[address] === ""
+        ? formatted
+        : state.aliases[address] ?? formatted,
+    [state, address, formatted]
+  );
+
+  return <span className={className}>{toDisplay}</span>;
```

Comment:
![image](https://user-images.githubusercontent.com/115862/223613696-dacac9eb-9713-4802-8014-ea04aaea264a.png)
I think the address should be copiable, like the one in sidebar, otherwise, no way to found out what exactly it is.